pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Description
+
+Please include a description of what was changed
+
+## Type of change
+
+- [ ] fix
+- [ ] feat
+- [ ] test
+- [ ] refactor
+- [ ] docs
+
+## Checklist
+
+- [ ] code has been self reviewed
+- [ ] code runs without any errors
+- [ ] thorough testing has been implemented if adding feature
+- [ ] all tests pass
+
+### Thanks!
```

Comment:
This looks great! Thank you for setting this up, it will really support with us having consistent PR's.

Reply:
Looks cool, we'll put it to good use.